diagnostic.md

```diff
@@ -6,21 +6,21 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    That is where the routing is done. Defining routes and such.
     ```
```

Comment:
We specify route definitions inside the Application Router.

In a Route, we specify how to load model data using the model hook.


That's the answer for the first part of it but there's more to it than that. 1/2